changelog/fix_false_positives_for_operator_method_call.md

```diff
@@ -0,0 +1 @@
+* [#13218](https://github.com/rubocop/rubocop/pull/13218): Fix false positives for `Style/OperatorMethodCall` with named forwarding. ([@earlopain][])
```

Comment:
```suggestion
* [#13224](https://github.com/rubocop/rubocop/pull/13224): Fix false positives for `Style/OperatorMethodCall` with named forwarding. ([@earlopain][])
```

Reply:
:roll_eyes:  I _really_ want this checked in CI, somehow